pandas/core/categorical.py

```diff
@@ -1340,8 +1351,14 @@ def _repr_categories_info(self):
         """ Returns a string representation of the footer."""
 
         category_strs = self._repr_categories()
-        levheader = "Categories (%d, %s): " % (len(self.categories),
-                                               self.categories.dtype)
+
+        from pandas.tseries.period import PeriodIndex
```

Comment:
OK, how about using ``inferred_type`` for ``PeriodIndex``, and ``dtype`` for others (for numpy compat).